nixos/tests/stanchion.nix

```diff
@@ -0,0 +1,29 @@
+import ./make-test.nix {
+  name = "stanchion";
+
+  nodes = {
+    master =
+      { pkgs, config, ... }:
+
+      {
+        services.riak.enable = true;
+        services.riak.package = pkgs.riak;
+
+        services.stanchion.enable = true;
+        services.stanchion.riakHost = "127.0.0.1:8087";
+        services.stanchion.listener = "127.0.0.1:8085";
+      };
+  };
+
+  testScript = ''
+    startAll;
+
+    $master->waitForUnit("riak");
+    $master->sleep(20); # Hopefully this is long enough!!
```

Comment:
Ah, sorry. Systemd does not work with Riak/Stanchion/Riak CS. @LnL7 and I attempted to solve this a few months back, but to no avail.
This is performed similarly to Riak's test.